src/main/java/duke/Type.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+/**
+ * Represents Enumeration class for types of commands.
+ */
+public enum Type {
+    TODO {
+        /**
+         * Returns string representation of ToDo task.
+         *
+         * @return String representation of ToDo task.
+         */
+        public String toString() {
+            return "ToDo";
+        }
+    }, DEADLINE {
+        /**
+         * Returns string representation of Deadline task.
+         *
+         * @return String representation of Deadline task.
+         */
+        public String toString() {
+            return "Deadline";
+        }
+    }, EVENT {
+        /**
+         * Returns string representation of Event task.
+         *
+         * @return String representation of Event task.
+         */
+        public String toString() {
+            return "Event";
+        }
+    }
+}
```

Comment:
Perhaps consider adding a toString method instead and pass the Strings as values? An example can be found with the Planet enum class at https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html